### PR TITLE
fix(DB/creature_template_addon): Add 'Shadowform' aura to Saronite Animus

### PR DESCRIPTION
**Pull request by Voxstrasza**
## CHANGES PROPOSED:
-  Apply a Shadowform Aura (Spell ID: 63359) to the Saronite Animus (10/25 player) when spawned during the General Vezax encounter.  I am 99% certain this is the effect required to give it the blizzlike look - I compared the spell aura added to plenty of screenshots from retail and it's a match.

## ISSUES ADDRESSED:
- Currently, Saronite Animus spawns as its base model, a white blob, without an aura to change the color.

## TESTS PERFORMED:
- image

[See this video to confirm](https://youtu.be/BUgBy65lVf4?t=190) that the portrait is supposed to be a white blob, but the NPC has a shadowform aura.

## HOW TO TEST THE CHANGES:
1.  Apply SQL
2.  Run Ulduar, make your way to General Vezax
3.  Pull boss and wait for Vezax to spawn to activate hardmode

## Target branch(es):
- [x] Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1606636935711383900.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606636935711383900');
+
+-- Apply 'Shadowform' Aura to Saronite Animus
+
+UPDATE `creature_template_addon` SET `auras` = '63359' WHERE (`entry` = 33524);
+UPDATE `creature_template_addon` SET `auras` = '63359' WHERE (`entry` = 34152);
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1606636935711383900.sql:6]: ints

```suggestion
UPDATE `creature_template_addon` SET `auras` = 63359 WHERE (`entry` = 33524);
UPDATE `creature_template_addon` SET `auras` = 63359 WHERE (`entry` = 34152);
```

**Review by Si1ker — CHANGES_REQUESTED** [review]

**Review by Si1ker — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1606636935711383900.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1606636935711383900.sql]: actually you are wrong silker

This table is a string so it should have quotes.

**Review by Kitzunu — DISMISSED** [review]
Change 'em back :P

**Review by Si1ker — APPROVED** [review]

**Comment by BarbzYHOOL** [comment]
this spell does this:

Apply Aura: DummyValue: -20

Dunno what it means 
Maybe is there a similar spell that does strictly nothing?

File: data/sql/updates/pending_db_world/rev_1606636935711383900.sql (hunk 1 above)
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1606636935711383900.sql:6]:
```suggestion
UPDATE `creature_template_addon` SET `auras` = '63359' WHERE `entry` IN (33524,34152);
```

**Review by BarbzYHOOL — DISMISSED** [review]
one query to rule them all

**Pull request merged**